install.rb

```diff
@@ -294,6 +308,16 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    if $operatingsystem == "windows"
+      localedir = File.join(Dir::COMMON_APPDATA, "PuppetLabs", "puppet", "share", "puppet", "locale")
```

Comment:
If we want to put this data in `Program Files` instead of `ProgramData`, what is the best way to tell if we should be using `(x86)` or not?